src/TestSuite/Constraint/Response/ContentType.php

```diff
@@ -43,6 +43,6 @@ public function matches($other)
      */
     public function toString()
     {
-        return 'was set as the Content-Type';
+        return 'was set as the Content-Type (`' . $this->response->getType() . '`)';
```

Comment:
```suggestion
        return 'is set as the Content-Type (`' . $this->response->getType() . '`)';
```

Other constraints uses present time.

Reply:
I guess I now need to fix some tests :)

Reply:
I updated some other past tense too now for consistency.